src/Model/PaymentCard.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace ThePay\ApiClient\Model;
+
+use DateTime;
+use ThePay\ApiClient\Utils\Json;
+
+class PaymentCard
+{
+    public const TYPE_CREDIT = 'credit';
+    public const TYPE_DEBIT = 'debit';
+    public const TYPE_PREPAID = 'prepaid';
+    public const TYPE_UNKNOWN = 'unknown';
+
+    private string $number;
+    private ?DateTime $expiration_date;
+    private ?string $brand;
+    private string $type;
+
+    /**
+     * @param string|array<string,string> $values Json in string or associative array
```

Comment:
U can use array shape `array{number: string, expiration_date: ?string,...}` as well in toArray method.